### ISSUE TITLE
Videos cannot be downloaded, because IG started blocking it.  (why) ?

### ISSUE THREAD

**Issue by xcientist** [HuiiBuh/InstagramDownloader#306]
![image](https://user-images.githubusercontent.com/49990037/172048957-f52d69d1-d4bd-4e54-8a4b-4bbe998873a0.png)

**Comment by mmagdy88** [comment]
+1

**Comment by kevinwinsper** [comment]
This appears to be working now when I tried a min ago.

**Comment by HuiiBuh** [comment]
Well,  I think the title sums it up pretty good.
**video download has been blocked by Instagram**

Why? Because they do not want you downloading stuff...

**Comment by kizo2703** [comment]
Now you can download SOME videos and stories, mostly in lo-res as I can see. But as soon you try to download something in high resolution, Insta is blocking it.

**Comment by Nissemus** [comment]
Not that this helps, but I don't really understand why IG are so against people downloading stuff. It's not like they own any of the content.

**Comment by mahye82** [comment]
This can't be accurate, since there are other websites and extensions that still have video downloading functionality. 

igram.io and Video & Photo Downloader For IG both do video downloading. They are imperfect solutions though, since they don't have the other functionality this project did.

**Comment by kschwerin** [comment]
> Not that this helps, but I don't really understand why IG are so against people downloading stuff. It's not like they own any of the content.

No. The person POSTING it owns the rights to it. Just because something is put online it doesn't mean they are giving people free use to use it. I suggest you read up on laws. Do you want some random dude saving all the family images you share on Facebook just because you wanted to show others about your vacation?

**Comment by afterdelight** [comment]
yesh

**Comment by Manwithpants812** [comment]
> Well, I think the title sums it up pretty good. **video download has been blocked by Instagram**
> 
> Why? Because they do not want you downloading stuff...

Kind of a shame this is the dev's response, a really unhelpful message. Reading it you'd think Instagram is actively blocking the dev's extension or has implemented "anti video download circumventions". That's not the case.

Instagram changed how some of their APIs worked last month ago, it broke most if not all Instagram extension/programs. The dev of this extension just hasn't found the new way to download videos and didn't care to elucidate that or doesn't care about updating the extension.

To everyone, use other programs like JDownloader and instaloader, they've been updated to work with the new APIs and can in fact download videos.

**Comment by YarekTyshchenko** [comment]
@Manwithpants812 The source is right there, submit a patch if you can. The author clearly doesn't have time or motivation for maintaining the extension, which is incerdibly time and effort consuming.

**Comment by afterdelight** [comment]
thats lame man

**Comment by Luen** [comment]
> Instagram changed how some of their APIs worked last month ago, it broke most if not all Instagram extension/programs. The dev of this extension just hasn't found the new way to download videos and didn't care to elucidate that or doesn't care about updating the extension.

Please submit a pull request and/or donate. 
It works much better than complaining.

**Comment by Manwithpants812** [comment]
> > Instagram changed how some of their APIs worked last month ago, it broke most if not all Instagram extension/programs. The dev of this extension just hasn't found the new way to download videos and didn't care to elucidate that or doesn't care about updating the extension.
> 
> Please submit a pull request and/or donate. It works much better than complaining.

No one is complaining.

I'm simply explaining for those confused, as I was, by the extension's message and by the dev's lackluster explanation that Instagram is not blocking the extension or blocking videos from being downloaded like the extension's message said, they simply changed how their API worked. They're 2 different things. The error message is misleading and confusing.

Not sure why you and YarekTyshchenko are being rude. In any case, I hope my comment helped those also confused, like the submitter of this issue, by the extension's pop up misleading message that "IG started blocking video downloads" message

**Comment by borayeris** [comment]
On Firefox it works but on Chrome it needs an update.

**Comment by afterdelight** [comment]
> > > Instagram changed how some of their APIs worked last month ago, it broke most if not all Instagram extension/programs. The dev of this extension just hasn't found the new way to download videos and didn't care to elucidate that or doesn't care about updating the extension.
> > 
> > 
> > Please submit a pull request and/or donate. It works much better than complaining.
> 
> No one is complaining.
> 
> I'm simply explaining for those confused, as I was, by the extension's message and by the dev's lackluster explanation that Instagram is not blocking the extension or blocking videos from being downloaded like the extension's message said, they simply changed how their API worked. They're 2 different things. The error message is misleading and confusing.
> 
> Not sure why you and YarekTyshchenko are being rude. In any case, I hope my comment helped those also confused, like the submitter of this issue, by the extension's pop up message and that it helps a dev willing to fix the issue find how other programs fixed it.

thats lame man

**Comment by kschwerin** [comment]
> lazy developer, you didn't think about releasing a new patch, no problem, I wrote a new plugin myself, you can learn how I use APIs from there.

He's said repeatedly he's done with the project. He's not lazy just because he doesn't care about this project that I lay let's you steal others images/videos. 

Being a dick will not earn you any respect so I suggest you stop.

**Comment by afterdelight** [comment]
its not stealing man, some people just want to fap

**Comment by afterdelight** [comment]
> > lazy developer, you didn't think about releasing a new patch, no problem, I wrote a new plugin myself, you can learn how I use APIs from there.
> 
> He's said repeatedly he's done with the project. He's not lazy just because he doesn't care about this project that Iets let's you steal others images/videos.
> 
> Being a dick will not earn you any respect so I suggest you stop.

whats your plugin link?

**Comment by afterdelight** [comment]
> lazy developer, you didn't think about releasing a new patch, no problem, I wrote a new plugin myself, you can learn how I use APIs from there.

whats your plugin link?
